firmware/Cargo.toml

```diff
@@ -205,8 +205,8 @@ fugit = "0.3"
 firmware_protocol = { path = "../networking/firmware_protocol", features = [
   "nalgebra031",
 ] }
-load-dotenv = "0.1"
 paste = "1.0"
+load-dotenv = "0.1"
```

Comment:
why not use [``dotenvy_macro``](https://crates.io/crates/dotenvy_macro)?

Reply:
I believe it requires you to specify the env var you want to check, but we actually want to load *all* env vars from .env